scripts/prepack.sh

```diff
@@ -4,9 +4,24 @@ set -euo pipefail
 shopt -s globstar
 
 # cross platform `mkdir -p`
-node -e 'fs.mkdirSync("build/contracts", { recursive: true })'
+mkdirp() {
+  node -e "fs.mkdirSync('$1', { recursive: true })"
+}
 
+# cd to the root of the repo
+cd "$(git rev-parse --show-toplevel)"
+
+npm run clean
+
+env COMPILE_MODE=production npm run compile
+
+mkdirp build/contracts
 cp artifacts/contracts/**/*.json build/contracts
 rm build/contracts/*.dbg.json
 
 node scripts/remove-ignored-artifacts.js
+
+cp README.md contracts/
+
+mkdirp contracts/build/contracts
+cp -r build/contracts/*.json contracts/build/contracts
```

Comment:
Here, the artifacts are copied twice, to `build/contracts` and `contracts/build/contracts`.

It looks like this is a leftover from when we had 2 packages. Maybe we should simplify that

Reply:
Fixed in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4628/commits/f63cf2da50589f776a2f0eabf045c2b679d6a3ca